internal/controller/statusreport/statusreport_adapter.go

```diff
@@ -300,6 +346,51 @@ func (a *Adapter) ReportSnapshotStatus(testedSnapshot *applicationapiv1alpha1.Sn
 	return nil
 }
 
+// ReportGroupSnapshotCreationStatus report the group snapshot creation status back to the git provider according the filtered integration test scenarios
+func (a *Adapter) ReportGroupSnapshotCreationStatus(snapshot *applicationapiv1alpha1.Snapshot, integrationTestScenarios *[]v1beta2.IntegrationTestScenario,
```

Comment:
Can you let me know in few sentences, why are we using integration test scenarios for getting the status of group snapshots?

Reply:
Do you mean when reporting status according to integration test scenario? This is to show the failure to git provider when failing to group snapshot on UI then users can know it without checking annotation.